swagger.yaml

```diff
@@ -959,6 +992,53 @@ definitions:
             type: string
             description: 9 digits SSN
             example: '123456789'
+  PhonesResultObject:
+    properties:
+      data:
+        type: array
+        items:
+          properties:
+            id:
+              type: string
+              description: Globally unique Id for the phone numbers
+            type:
+              type: string
+              example: phone
+            attributes:
+              properties:
```

Comment:
Descriptions of some of these attributes are missing. Can you add descriptions to all of them?